mirall.desktop.in

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env xdg-open
```

Comment:
?

Reply:
I did this by accident. Sorry, will undo it. :)